### PR TITLE
Link to forum

### PR DESCRIPTION
**Pull request by matthiasgeiger**
Adds another entry to the "Help" menu: "Online help forum" (see https://github.com/JabRef/jabref/issues/1521#issuecomment-230994579)

- [x] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Screenshots added (for bigger UI changes)

### CODE REVIEW

File: src/main/java/net/sf/jabref/gui/actions/OpenBrowserAction.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import javax.swing.AbstractAction;
+import javax.swing.Action;
+import javax.swing.Icon;
+
+import net.sf.jabref.gui.desktop.JabRefDesktop;
+
+public class OpenBrowserAction extends AbstractAction {
+
+    private String urlToOpen;
```
Comment (oscargus) [inline src/main/java/net/sf/jabref/gui/actions/OpenBrowserAction.java]: final

**Comment by oscargus** [comment]
LGTM!

File: src/main/java/net/sf/jabref/gui/actions/OpenBrowserAction.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+import java.awt.event.ActionEvent;
+
+import javax.swing.AbstractAction;
+import javax.swing.Action;
+import javax.swing.Icon;
+
+import net.sf.jabref.gui.desktop.JabRefDesktop;
+
+public class OpenBrowserAction extends AbstractAction {
```
Comment (tobiasdiez) [inline src/main/java/net/sf/jabref/gui/actions/OpenBrowserAction.java]: I think the UpdateDialog also has some actions which might be replaced by this one.

Comment (matthiasgeiger) [inline src/main/java/net/sf/jabref/gui/actions/OpenBrowserAction.java]: No. Only the `JabRefDesktop.openBrowserShowPopup(...)` method is used there - but not by an `AbstractAction` (which is used for menus and icon bars only).

**Comment by stefan-kolb** [comment]
LGTM :+1:

**Pull request merged**